### PR TITLE
Add tests for menu dispatches

### PR DESCRIPTION
**Pull request by captainsafia**
Testing the untested.

### CODE REVIEW

**Comment by jdetle** [comment]
We can't rely on coveralls for great coverage reports but we can rely on @captainsafia for great tests!

**Comment by captainsafia** [comment]
> We can't rely on coveralls for great coverage reports but we can rely on @captainsafia for great tests!

You'e too kind!

File: src/notebook/menu.js [hunk 1]
```diff
@@ -232,7 +232,7 @@ export function dispatchPasteCell(store) {
 export function dispatchCreateCellAfter(store) {
   const state = store.getState();
   const focused = state.document.get('focusedCell');
-  store.dispatch(createCellAfter(focused));
+  store.dispatch(createCellAfter('code', focused));
```
Comment (rgbkrk) [inline src/notebook/menu.js:235]: Love it when bugs are uncovered in testing. :smile:

File: test/setup.js [hunk 2]
```diff
@@ -48,4 +48,13 @@ mock('electron', {
...
   },
+  'remote': {
+    'require': function(module) {
+      if (module === 'electron') {
+        return {
+          'dialog': function() { },
+        };
+      }
+    },
+  },
```
Comment (rgbkrk) [inline test/setup.js:59]: Slick. :bow:

**Comment by rgbkrk** [comment]
Such a great suite of tests, thanks @captainsafia!

**Pull request merged**